test/integration/fakeapp/config/boot.rb

```diff
@@ -0,0 +1,12 @@
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
+
+require 'bundler/setup' # Set up gems listed in the Gemfile.
+require 'bootsnap' # Speed up boot time by caching expensive operations.
+Bootsnap.setup(
+  cache_dir: 'tmp/cache',
+  development_mode: true,
```

Comment:
forcing this to be true at least makes travis output consistent with local... still not great